modules/distr-tracing-tempo-config-multitenancy.adoc

```diff
@@ -9,6 +9,11 @@
 Multitenancy with authentication and authorization is provided in the Tempo Gateway service.
 The authentication uses OpenShift OAuth and the Kubernetes `TokenReview` API. The authorization uses the Kubernetes `SubjectAccessReview` API.
 
+[NOTE]
+====
+The Tempo Gateway service supports ingestion of traces only via the OTLP protocol over the gRPC framework. The OTLP protocol is not supported over the HTTP protocol.
```

Comment:
I think it sounds a bit weird. Maybe we should just refer it as `OTLP/gRPC` and `OTLP/HTTP`?
Here are the upstream docs: https://opentelemetry.io/docs/specs/otlp/

```suggestion
The Tempo Gateway service supports ingestion of traces only via OTLP/gRPC. OTLP/HTTP is not supported.
```

Reply:
@andreasgerstmayr, let's try it like this. I hope the peer reviewer won't have any issues with this.